tests/multi_net/tcp_accept_recv.py

```diff
@@ -11,13 +11,14 @@ def instance0():
     s = socket.socket()
     s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
     s.bind(socket.getaddrinfo("0.0.0.0", PORT)[0][-1])
-    s.listen(1)
+    s.listen()
```

Comment:
This might change the original test with 1 as an argument.

Maybe it's better to do a loop here to perform both tests (with no argument, and with an argument of 1, and then maybe some other values like 0 for good measure).

Also need the client to do a loop as well, and use `multitest.wait()` and `multitest.broadcast()` to sync with the server.

Reply:
Ok, I've got an updated looping test just pushed that passes on the pyb-d with this patch applied.